gatsby-node.js

```diff
@@ -19,7 +19,12 @@ exports.createPages = ({ graphql, actions }) => {
               filter: {
                 fields: {
                   slug: {
-                    nin: ["", "nodejs-community", "homepage", "privacy-policy"]
+                    nin: [
+                      ""
+                      "nodejs-community"
+                      "homepage"
+                      "trademark-policy"
```

Comment:
Your latest merge removed `privacy-policy` from this array. I think you need to add it again.

Reply:
Thank you!!